src/screens/LoginScreen/index.tsx

```diff
@@ -84,7 +84,11 @@ const LoginScreen: FC = () => {
             <Image mt={5} alt="CMI logo" size="sm" source={whiteLogo} />
           </Center>
 
-          {/*<Pressable onPress={() => {}}>/!*<Icon as={FontAwesome} name="database" />*!/</Pressable>*/}
+          <Flex mr={4} mb={6} alignItems="flex-end">
+            <Pressable onPress={navigateToAppLanguage}>
+              <Icon name="database" size={22} color="#fff" />
```

Comment:
Please create a separate component for the icon and put it inside "@shared/ui/icons".